### PR TITLE
Resolution of a few self-reference entries in bhnd_erom.9 device_add_child.9 cfumass.4

### PR DESCRIPTION
**Pull request by chrisdavidson**
bhnd_erom.9 - fixed self reference entry .Xr -> .Nm
device_add_child.9 - fixed self reference entry .Xr -> .Nm
cfumass.4 - fixed self reference entry .Xr -> .Nm

### CODE REVIEW

File: lib/libsys/open.2 [hunk 1]
```diff
@@ -323,7 +323,7 @@ by the descriptor at the time of the call.
 .Pp
 .Dv O_PATH
 returns a file descriptor that can be used as a directory file descriptor for
-.Xr openat 2
+.Nm openat 2
```
Comment (bsdimp) [inline lib/libsys/open.2:326]: .Fn openat

File: share/man/man4/cfumass.4 [hunk 2]
```diff
@@ -57,7 +57,7 @@ To use
 .Nm :
 .Bl -bullet
 .It
-.Xr cfumass 4
+.Nm cfumass 4
```
Comment (bsdimp) [inline share/man/man4/cfumass.4:60]: .Nm cfumass

might be better. Or maybe even just .Nm

File: share/man/man9/device_add_child.9 [hunk 3]
```diff
@@ -111,10 +111,10 @@ When adding a child to another device node, such as in an identify
 routine, use
 .Xr BUS_ADD_CHILD 9
 instead of
-.Xr device_add_child 9 .
+.Nm device_add_child 9 .
```
Comment (bsdimp) [inline share/man/man9/device_add_child.9:114]: .Fn device_add_child .

File: share/man/man9/device_add_child.9 [hunk 4]
```diff
@@ -111,10 +111,10 @@ When adding a child to another device node, such as in an identify
...
 .Xr BUS_ADD_CHILD 9
 will call
-.Xr device_add_child 9
+.Nm device_add_child 9
```
Comment (bsdimp) [inline share/man/man9/device_add_child.9:117]: .Fn device_add_child

(sorry if I mislead you)

**Comment by bsdimp** [comment]
I think my advice on discord might have been wide of the mark.

File: lib/libsys/open.2 (hunk 1 above)
Comment (chrisdavidson) [reply lib/libsys/open.2]: resolved in the latest commit

File: share/man/man9/device_add_child.9 (hunk 3 above)
Comment (chrisdavidson) [reply share/man/man9/device_add_child.9]: resolved in latest commit

**Comment by chrisdavidson** [comment]
this pull request is now in a much better state than it was before. a rebase was performed to consolidate changes into one commit instead of 8 commits.

**Review by concussious — CHANGES_REQUESTED** [review]
There's an extra space at the end of line 326

**Pull request closed without merging**